apps/opik-backend/src/main/java/com/comet/opik/domain/DatasetItemDAO.java

```diff
@@ -513,12 +513,24 @@ private String getOrDefault(UUID value) {
     private Publisher<DatasetItem> mapItem(Result results) {
         return results.map((row, rowMetadata) -> {
 
-            Map<String, JsonNode> data = getData(row);
+            Map<String, JsonNode> data = new HashMap<>(getData(row));
```

Comment:
Minor: no need to wrap it up around another HashMap. It seems that the `getData` method already returns a local mutable map, so no risks here.